csu_blended_rain_tropical.py

```diff
@@ -0,0 +1,237 @@
+"""
+csu_blended_rain.py
+
+# Brody Fuchs, CSU, Oct 2014
+# brfuchs@atmos.colostate.edu
+
+# python remake of Brenda's code to calculate water and ice mass
+
+Amendments by
+Timothy Lang (tjlangco@gmail.com)
+2/20/2015
+
+"""
+
+import numpy as np
+import warnings
+from csu_liquid_ice_mass import linearize, get_linear_fits
+
+
+def calc_rain_zr(dz, a=216, b=1.39, band='S'):
+    """
+    dz = Reflectivity (dBZ), returns rainfall rate (mm h^-1)
+    Form Z = a * R**b
+    """
+    return (linearize(dz) / a)**(1.0 / b)
+
+
+def calc_trop_rain_kdp_zdr(
+    kdp,
+    zdr,
+    a=96.5726,
+    b=0.315,
+    c=-2.1140,
+    band='S',
+        predef='True'):
+    """
+    kdp = Specific Differential Phase (deg km^-1)
+    zdr = Differential Reflectivity (dB)
+    a, b, c = Adjustable function parameters
+    """
+
+    if predef == 'True':
+        if band == 'S':
+            a = 96.5726
+            b = 0.315
+            c = -2.1140
+
+        if band == 'C':
+            a = 45.6976
+            b = 0.8763
+            c = -1.6718
+
+        if band == 'X':
+            a = 28.1289
+            b = 0.9194
+            c = -1.6876
+
+    return a * kdp**b * linearize(zdr)**c
+
+
+def calc_trop_rain_z_zdr(
+    dz,
+    zdr,
+    a=0.0085,
+    b=0.9237,
+    c=-5.2389,
+    band='S',
+        predef='True'):
+    """
+    dz = Reflectivity (dBZ)
+    zdr = Differential Reflectivity (dB)
+    a, b, c = Adjustable function parameters
+    """
+    if predef == 'True':
+        if band == 'S':
+            a = 0.0085
+            b = 0.9237
+            c = -5.2389
+
+        if band == 'C':
+            a = 0.0086
+            b = 0.9088
+            c = -4.2059
+
+        if band == 'X':
+            a = 0.0085
+            b = 0.9294
+            c = -4.4580
+
+    return a * linearize(dz)**b * linearize(zdr)**c
+
+
+def calc_trop_rain_kdp(kdp, a=59.5202, b=0.7451, band='S', predef='True'):
+    """
+    kdp = Specific Differential Phase (deg km^-1)
+    a, b = Adjustable coefficient, exponent
+
+    The coefficients below are from Thompson et al. 2016. They are for the
+    convective regime since Kdp has to be significant enough to use, which is
+    essentially for convective points.
+    """
+    if predef == 'True':
+        if band == 'S':
+            a = 59.5202
+            b = 0.7451
+        if band == 'C':
+            a = 34.5703
+            b = 0.7331
+        if band == 'X':
+            a = 21.9729
+            b = 0.7221
+
+    return a * kdp**b
+
+
+def calc_blended_rain_tropical(

```

Comment:
Should add an import of `calc_blended_rain_tropical` to the csu_blended_rain.py file, so this function can become part of the overall `csu_blended_rain` sub-module.